commit 747bb61652c5563cc10cbcaa755ea1b14ccf6482
Author: Kyle Scheuing <kyle@kylescheuing.com>
Date:   Mon Nov 27 19:22:59 2023 -0500

    docs(gui): add screenshots to appstream metadata
    
    We previously only had a demo video, but no screenshots.

diff --git a/gui/resources/com.oppzippy.OpenSCQ30.metainfo.xml b/gui/resources/com.oppzippy.OpenSCQ30.metainfo.xml
index 30dd214..291f4e2 100644
--- a/gui/resources/com.oppzippy.OpenSCQ30.metainfo.xml
+++ b/gui/resources/com.oppzippy.OpenSCQ30.metainfo.xml
@@ -25,6 +25,14 @@
     <content_rating type="oars-1.0" />
     <launchable type="desktop-id">com.oppzippy.OpenSCQ30.desktop</launchable>
     <screenshots>
+        <screenshot>
+            <image>https://raw.githubusercontent.com/Oppzippy/OpenSCQ30/dceaeaf4556ba7ef2096ab67948384a3deef8a16/screenshots/gui_general.png</image>
+            <caption>General Settings</caption>
+        </screenshot>
+        <screenshot>
+            <image>https://raw.githubusercontent.com/Oppzippy/OpenSCQ30/dceaeaf4556ba7ef2096ab67948384a3deef8a16/screenshots/gui_equalizer.png</image>
+            <caption>Equalizer Settings</caption>
+        </screenshot>
         <screenshot>
             <video container="webm" codec="vp9" width="902" height="386">https://user-images.githubusercontent.com/2236514/229958756-aaa2a6d3-e908-4195-aad6-b0bcfda139a8.webm</video>
         </screenshot>
@@ -38,7 +46,7 @@
                 <p>Features</p>
                 <ul>
                     <li>Partial support for A3945, A3933, and A3939. Not ready for general use.</li>
-                </ul> 
+                </ul>
             </description>
         </release>
         <release version="1.8.1" date="2023-10-31" />
